### ISSUE TITLE
Play media over browser_mod does not work because of mixed content HTTP/HTTPS

### ISSUE THREAD

**Issue by rchovan** [thomasloven/hass-browser_mod#301]
My Home Assistant version: 2022.2.9, plugin installed over HACS.

**What I am doing:**
Play media over browser_mod does not work because of mixed content. HTTP connection is upgraded to HTTPS, but lot of stream servers does not use HTTPS.

**What I expected to happen:**
Play URL stream

**What happened instead:**
Nothing, but error is logged in console. 

**Minimal** steps to reproduce:
Play URL HTTP stream on HA where you are connected over HTTPS.
Example code from developer tools > services

```yaml
# The least amount of code or steps possible to reproduce my error
service: media_player.play_media
data:
  media_content_type: music
  media_content_id: http://live.slovakradio.sk:8000/Slovensko_128.mp3
target:
  device_id: 803097b77317c14289294dfd2eaad0ec
# Put your code/steps here
```

**Error messages from the browser console:**

```
// Select everything from the browser console and copy it
Mixed Content: The page at 'https://<redacted>/lovelace-music/0' was loaded over HTTPS, but requested an insecure element 'http://live.slovakradio.sk:8000/Slovensko_128.mp3'. This request was automatically upgraded to HTTPS, For more information see https://blog.chromium.org/2019/10/no-more-mixed-messages-about-https.html
2browser_mod.js:145 Uncaught (in promise) DOMException: The play() request was interrupted by a new load request. https://goo.gl/LdLk22
player_play @ browser_mod.js:145
play @ browser_mod.js:145
msg_callback @ browser_mod.js:145
_connection.subscribeMessage.type @ browser_mod.js:145
_handleMessage @ core.30819411.js:1
live.slovakradio.sk:8000/Slovensko_128.mp3:1          Failed to load resource: net::ERR_CONNECTION_CLOSED
/lovelace-music/0:1 Uncaught (in promise) DOMException: Failed to load because no supported source was found.
DevTools failed to load source map: Could not load content for chrome-extension://hmigninkgibhdckiaphhmbgcghochdjc/contentScript.js.map: HTTP error: status code 404, net::ERR_UNKNOWN_URL_SCHEME
// Paste it here
```

---

**By replacing the space in the checkboxes ([ ]) with an X below, I indicate that I:**

- [X] Understand that this is a channel for reporting bugs, not a support forum (https://community.home-assistant.io/).

- [X] Have made sure I am using the latest version of the plugin.

- [X] Have followed the troubleshooting steps of the "Common Problems" section of https://github.com/thomasloven/hass-config/wiki/Lovelace-Plugins.

- [X] Understand that leaving one or more boxes unticked or failure to follow the template above may increase the time required to handle my bug-report, or cause it to be closed without further action.

**Comment by rchovan** [comment]
Hi,  any news or progress ? Or do you know any workaround ?

**Comment by rchovan** [comment]
Any progress?

**Comment by thomasloven** [comment]
Closed because of Browser Mod 2.0 release.
Please see #354.
